Sprint-2/debug/address.js

```diff
@@ -12,4 +14,4 @@ const address = {
   postcode: "XYZ 123",
 };
 
-console.log(`My house number is ${address[0]}`);
+console.log(`My house number is ${address["houseNumber"]}`);
```

Comment:
This is correct. There is an even cleaner notation however, could you point it out?

Reply:
Thank you for review, yes, there a cleaner and more concise way to access the property of the object would be using dot notation like this:
console.log(`My house number is ${address.houseNumber}`);